src/StreamrClient.ts

```diff
@@ -18,21 +18,29 @@ import { DataUnion, DataUnionDeployOptions } from './dataunion/DataUnion'
 import { BigNumber } from '@ethersproject/bignumber'
 import { getAddress } from '@ethersproject/address'
 import { Contract } from '@ethersproject/contracts'
+import { StreamPartDefinition } from './stream'
 
 // TODO get metadata type from streamr-protocol-js project (it doesn't export the type definitions yet)
 export type OnMessageCallback = MaybeAsync<(message: any, metadata: any) => void>
 
+// TODO check if these are correct
+export interface SubscribeOptions {
+    resend?: boolean
+    from?: { timestamp: number, sequenceNumber?: number }
+    to?: { timestamp: number, sequenceNumber?: number }
+    last?: number
+}
```

Comment:
`resend` is not a boolean.

basically, resend options can either be nested or top-level in the config.

```typescript
type ResendOptions = {
    from?: { timestamp: number, sequenceNumber?: number }
    to?: { timestamp: number, sequenceNumber?: number }
    last?: number
}

type SubscribeOptions = {
    resend?: ResendOptions
} & ResendOptions
```

Reply:
Fixed